variables.tf

```diff
@@ -103,6 +109,12 @@ variable "service_credential_names" {
   }
 }
 
+variable "member_enabled" {
```

Comment:
This variable should not be needed. I think it's making the interface of the module more complicated than it ought to be. Could we set the member variable to 1 when the variable remote_leader_crn is set in the main.tf?

Reply:
Removed the extra variable and added the condition for dynamic member using `remote_leader_crn`.